### PR TITLE
Buck dev

### PR DESCRIPTION
**Pull request by only1buckie**
Check implementations for Task

### CODE REVIEW

File: src/main/java/com/bucky/employee/controller/TaskController.java [hunk 1]
```diff
@@ -39,31 +40,40 @@ public class TaskController {
...
     
      @GetMapping("/searchbytaskname")
-    public Task findTaskByName(@RequestParam("taskname") String taskName) {
+    public ResponseEntity <Task> findTaskByName(@RequestParam("taskname") String taskName) {
 
         Task task = taskService.findByTaskName(taskName);
-        return task;
+        HttpHeaders header = new HttpHeaders();
+        header.add("Get Task By name","Found task");
+        return ResponseEntity.status(HttpStatus.OK).headers(header).body(task);
```
[the remark this answers is not in the document]
Comment (only1buckie) [reply src/main/java/com/bucky/employee/controller/TaskController.java:67]: Yes

File: src/main/java/com/bucky/employee/controller/TaskController.java [hunk 2]
```diff
@@ -72,23 +82,32 @@ public List<Task> getAllTasks() {
...
 
     @DeleteMapping("/deletetask")
-    public Task deleteById(@RequestParam("id") Long Id) {
+    public ResponseEntity <Task> deleteById(@RequestParam("id") Long Id) {
         Task task = taskService.deleteTaskById(Id);
-        return task;
+
+        HttpHeaders header = new HttpHeaders();
+        header.add("Delete Task","Task Deleted");
+        return ResponseEntity.status(HttpStatus.OK).headers(header).body(task);
```
[the remark this answers is not in the document]
Comment (only1buckie) [reply src/main/java/com/bucky/employee/controller/TaskController.java:90]: Done

File: src/main/java/com/bucky/employee/controller/TaskController.java [hunk 3]
```diff
@@ -72,23 +82,32 @@ public List<Task> getAllTasks() {
...
     
     @PutMapping("/updatetask")
-    public Task updateTask(@RequestBody Task input){
+    public ResponseEntity <Task> updateTask(@RequestBody Task input){
     
         Task task = taskService.updateTask(input);
-        return task;
+        HttpHeaders header = new HttpHeaders();
+        header.add("Update Task","Task Updated");
+        return ResponseEntity.status(HttpStatus.OK).headers(header).body(task);
```
[the remark this answers is not in the document]
Comment (only1buckie) [reply src/main/java/com/bucky/employee/controller/TaskController.java:99]: In postman there's a header tab with various fields of information, just extra orishirishi

[the remark this answers is not in the document]
Comment (audreySuz) [reply src/main/java/com/bucky/employee/controller/TaskController.java:99]: Please take out unnecessary implementations

[the remark this answers is not in the document]
Comment (only1buckie) [reply src/main/java/com/bucky/employee/controller/TaskController.java:99]: Okay

**Pull request merged**